src/db/models/articleCategory.js

```diff
@@ -0,0 +1,21 @@
+export default (sequelize, DataTypes) => {
+  const ArticleCategory = sequelize.define(
+    'ArticleCategory',
+    {
+      authorId: DataTypes.UUID,
+      categoryId: DataTypes.UUID
+    }, {}
```

Comment:
```suggestion
    },
    {}
```
Please be consistent in your indentations. Either us this style or use the one you used for the Author Model

Reply:
Okay. Done!